Furball.Vixie/Graphics/Backends/OpenGL20/LineRendererGL20.cs

```diff
@@ -14,9 +15,9 @@ private struct LineData {
         
         private readonly OpenGL20Backend _backend;
 
-        private const int BATCH_MAX = 128; //cut this in two for actual line could, as we use 2 LineData structs per line :^)
+        private const int BATCH_MAX = 128; //cut this in two for actual line couyt, as we use 2 LineData structs per line :^)
```

Comment:
why did you fuck up perfectly good spelling

Reply:
i-- dont... know

Reply:
OH i know why, i tried to correct it from could -> count, but fucked up